Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -965,6 +965,12 @@ public final class SwiftModuleBuildDescription {
             break
         }
 
+        if bundlePath != nil {
+            compilationConditions += ["-DSWIFT_MODULE_RESOURCE_BUNDLE_AVAILABLE=1"]
+        } else {
+            compilationConditions += ["-DSWIFT_MODULE_RESOURCE_BUNDLE_UNAVAILABLE=1"]
```

Comment:
Won't this trigger

> warning: conditional compilation flags do not have values in Swift; they are either present or absent

Reply:
oh good catch! I was copying the wrong pattern from the Clang version.